Framework/Core/include/Framework/BasicOps.h

```diff
@@ -46,6 +49,61 @@ enum BasicOp : unsigned int {
   BitwiseNot,
   Conditional // 3-ar functions
 };
+
+static constexpr std::array<std::string_view, BasicOp::Conditional + 1> mapping{
```

Comment:
Do you really need to have this in the header? This will bloat everywhere it's included.

Reply:
Moved to .cxx